Adding docs page for eCloud VPC VPN's

hep147258 opened the pull request.

hep147258 committed:
adding first draft of vpn doc

hep147258 committed:
editing vpn session screenshot

hep147258 committed:
Editing VPN session photo again

hep147258 committed:
Changing VPN session image

hep147258 committed:
adding new VPN session image

hep147258 committed:
formatting tweak

hep147258 committed:
Adding more VPNs to table

hep147258 committed:
Adding IKE column to table

hep147258 committed:
grammer tweak

hep147258 committed:
meta data change

hep147258 committed:
removing trailing spaces

hep147258 committed:
Spelling corrections

CountBacula reviewed (approved):
LGTM

hep147258 committed:
Capitalization fix for IPSec

hep147258 committed:
Capitalization fix of ipsec

hep147258 committed:
adding words to wordlist

hep147258 committed:
Merge branch 'master' of https://github.com/hep147258/docs.ukfast.co.uk-1

hep147258 committed:
adding keywords

Xiol commented on source/ecloud/ecloud-vpc/vpn.md:
```diff
@@ -0,0 +1,78 @@
+
+There have been a range of VPN Profiles created, which have various levels of encryption settings. Details of these can be seen in the following table. If you need any advise on which VPN profiles to use, please get in touch with our support team.
+
+| Profile Name        | IKE Version | Phase 1 - Encryption | Phase 1 - Integrity | Phase 1 - DH Group | Phase 1 - PRF | Phase 1 - SA Lifetime | Phase 2 - Encryption | Phase 2 - Integrity | Phase 2 - SA Lifetime | Phase 2 - PFS |
```
This table is broken when rendered.

See https://pr-1002.docsuat.devops.ans.uk/ecloud/ecloud-vpc/vpn.html

hep147258 replied:
Will have a look at this and see how best to do it

Xiol commented on source/ecloud/ecloud-vpc/vpn.md:
```diff
@@ -0,0 +1,78 @@
+# eCloud VPC VPNs
```
This page isn't linked to the TOC for eCloud VPC, you'll need to add it in [here](https://github.com/ukfast/docs.ukfast.co.uk/blob/master/source/ecloud/ecloud-vpc/index.rst).

hep147258 replied:
Thanks @Xiol, I have added that now

hep147258 committed:
adding vpn to index

hep147258 committed:
changing table to rst

miff2000 committed:
Remove eCloud VPC from the page title

It's there in the breadcrumbs and content body, so it's clear already

miff2000 committed:
Split table into two to make it more readable

miff2000 committed:
list-table format

hep147258 committed:
changing table formatting

hep147258 committed:
fixing title for table

hep147258 committed:
removing list table

hep147258 committed:
moving title

miff2000 committed:
Shorten title for left menu

The tab title and metadata are taken from the block at the bottom of the document

miff2000 committed:
Update title
